flow/util/def2stream.py

```diff
@@ -101,7 +95,6 @@
             top.insert(pya.CellInstArray(cell.cell_index(), pya.Trans()))
 
 # Write out the GDS
-print("[INFO] Writing out GDS/OAS '{0}'".format(out_file))
```

Comment:
I think it is helpful to tell what file was written.

Reply:
This information is logged as the command line is logged.

This is consistent with e.g. cp, gcc, etc. not to repeat information tersely and accurately described by logging the command.

I dont mind making the change to the OR, but I think there is something to be said for the approach of cp, gcc, etc to log the command line and not add this logging.